pages/documentation/developer/tutorials/emf.md

```diff
@@ -0,0 +1,361 @@
+  </dependency>
+  <dependency>
+    <groupId>com.powsybl</groupId>
+    <artifactId>powsybl-cgmes-conversion</artifactId>
+    <version>${powsybl.core.version}</version>
+  </dependency>
+  <dependency>
+    <groupId>com.powsybl</groupId>
+    <artifactId>powsybl-cgmes-extensions</artifactId>
+    <version>${powsybl.core.version}</version>
+  </dependency>
+  <dependency>
+    <groupId>com.powsybl</groupId>
+    <artifactId>powsybl-commons</artifactId>
+    <version>${powsybl.core.version}</version>
+  </dependency>
+  <dependency>
+    <groupId>com.powsybl</groupId>
+    <artifactId>powsybl-iidm-converter-api</artifactId>
+    <version>${powsybl.core.version}</version>
+  </dependency>
+  <dependency>
+    <groupId>com.powsybl</groupId>
+    <artifactId>powsybl-iidm-mergingview</artifactId>
+    <version>${powsybl.core.version}</version>
+  </dependency>
+  <dependency>
+    <groupId>com.powsybl</groupId>
+    <artifactId>powsybl-open-loadflow</artifactId>
+    <version>${powsybl.olf.version}</version>
+  </dependency>
+
+  <dependency>
+    <groupId>com.powsybl</groupId>
+    <artifactId>powsybl-config-classic</artifactId>
+    <version>${powsybl.core.version}</version>
+    <scope>runtime</scope>
+  </dependency>
+  <dependency>
+    <groupId>com.powsybl</groupId>
+    <artifactId>powsybl-iidm-impl</artifactId>
+    <version>${powsybl.core.version}</version>
+    <scope>runtime</scope>
+  </dependency>
+  <dependency>
+    <groupId>com.powsybl</groupId>
+    <artifactId>powsybl-triple-store-impl-rdf4j</artifactId>
+    <version>${powsybl.core.version}</version>
+  </dependency>
+  <dependency>
+    <groupId>org.slf4j</groupId>
+    <artifactId>log4j-over-slf4j</artifactId>
+    <version>${slf4j.version}</version>
+    <scope>runtime</scope>
+  </dependency>
+  <dependency>
+    <groupId>ch.qos.logback</groupId>
+    <artifactId>logback-classic</artifactId>
+    <version>1.1.8</version>
+    <scope>runtime</scope>
+  </dependency>
+  <dependency>
+    <groupId>com.powsybl</groupId>
+    <artifactId>powsybl-entsoe-cgmes-balances-adjustment</artifactId>
+    <version>${powsybl.entsoe.version}</version>
+    <scope>compile</scope>
+  </dependency>
+</dependencies>
+```
+
+## Configure PowSyBl
+The configuration file for PowSyBl should be located in the folder `emf/initial/src/main/resources`. It gathers all the default configuration parameters about the load flow simulator, the parameters for the import/export of CGMES files, the path to the CGMES files, the path to the PEVF file, the output directory, etc.
+
+We need to configure PowSyBl to use the OpenLoadFlow implementation. To do that, you need to edit the `config.yml` file: 
+```yaml
+load-flow:
+  default-impl-name: "OpenLoadFlow"
+```
+The parameters for the import and export of CGMES files are, with the proper path to the folder containing the boundary files:
+```yaml
+import-export-parameters-default-value:
+  iidm.import.cgmes.boundary-location: path-to-EQBD-TPBD
```

Comment:
For fields to complete by the user, you can add a comment `# complete with the path to your CGMES boundary files folder`